src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -129,27 +137,56 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
   /** Stream of escape keyboard events. */
   private _escapeEventStream = new Subject<void>();
 
+  /** Emits once and only once on the next onStable event. */
+  private readonly _nextStable: Observable<void>;
+
+  /**  The current model value. */
+  private _value: any;
+
   /** View -> model callback called when value changes */
   _onChange: (value: any) => void = () => {};
 
   /** View -> model callback called when autocomplete has been touched */
   _onTouched = () => {};
 
   /* The autocomplete panel to be attached to this trigger. */
-  @Input('matAutocomplete') autocomplete: MatAutocomplete;
+  @Input('matAutocomplete')
+  set _autocomplete(value: MatAutocomplete) {
+    this.autocomplete = value;
+    if (this._displayWithSubscription) {
```

Comment:
Rather than null checking here, you can initialize the `_displayWithSubscription` to `Subscription.EMPTY`.

Reply:
Made this change. Looks a lot better.